src/Microsoft.AspNetCore.Mvc.Core/ActionConstraints/ActionConstraintCache.cs

```diff
@@ -7,12 +7,11 @@
 using System.Linq;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc.Abstractions;
-using Microsoft.AspNetCore.Mvc.ActionConstraints;
 using Microsoft.AspNetCore.Mvc.Infrastructure;
 
-namespace Microsoft.AspNetCore.Mvc.Internal
+namespace Microsoft.AspNetCore.Mvc.ActionConstraints
 {
-    public class ActionConstraintCache
+    internal sealed class ActionConstraintCache
```

Comment:
Adding `sealed` probably doesn't hurt. But, it doesn't help much with `internal` types and is added randomly in this PR. Suggest reverting all of these additions.

Reply:
If its just one find and replace, I would change it. That way in the future people will not wonder why they are internal sealed or think that this is an actual code pattern that we follow